spec/controllers/bundles_controller_spec.rb

```diff
@@ -93,9 +93,29 @@
           do_request
         end
 
-        it "gives the user the option to submit a request for approval" do
-          expect(assigns[:add_to_cart]).to be_blank
-          assert_redirected_to(new_facility_product_training_request_path(facility, bundle))
+        around(:each) do |example|
+          saved_setting = SettingsHelper.feature_on?(:training_requests)
+          SettingsHelper.enable_feature(:training_requests, training_requests)
+          example.run
+          SettingsHelper.enable_feature(:training_requests, saved_setting)
+        end
+
+        context "if the training request feature is enabled" do
+          let(:training_requests) { true }
```

Comment:
Can we make this variable something more like `training_requests_enabled` to make it a little more readable in the `around`?